lib/shopping_cart.rb

```diff
@@ -0,0 +1,22 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+  end
+
+  def capacity
+    @capacity.chomp("items")
+    @capacity.to_i
+  end
```

Comment:
similarly here, stick to either a method called capacity or an attr_reader for capacity... since you are formatting the return value, I'd stick to the method for capacity.